tests/functional/firefox/whatsnew/test_whatsnew_73.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from selenium.common.exceptions import TimeoutException
+from pages.firefox.whatsnew.whatsnew_73 import FirefoxWhatsNew73Page
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
```

Comment:
We should replace these tests with a single test for the default CTA, which doesn't depend on Firefox being set as the default browser by the OS.